Routes/users.route.js

```diff
@@ -0,0 +1,24 @@
+const express = require("express");
+const route = express.Router();
+
+route.get('/single/:id', (req, res, next) => {
```

Comment:
I think it should be /user/:id

Reply:
If it's user/:id it will mess with any other GET requests to /user (from my experience)